jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -326,7 +326,7 @@ protected void doFilter(HttpServletRequest request, HttpServletResponse response
         tracker = getRateTracker(request);
 
         // Calculate the rate and check if it is over the allowed limit
-        final OverLimit overLimit = tracker.isRateExceeded(System.currentTimeMillis());
+        final OverLimit overLimit = tracker.isRateExceeded(System.nanoTime());
```

Comment:
Can you update the javadoc of `isRateExceeded`. It specifies that the value is in ms.
>  now the time now (in milliseconds)